src/controllers/staff.controller.ts

```diff
@@ -1 +1,55 @@
-import { StaffModel } from "../models/staff";
+import { StaffModel, staffType } from "../models/staff";
+
+export const createStaff = async (staff: staffType) => {
+  const newStaff = new StaffModel(staff);
+  await newStaff.save();
+  return newStaff;
+};
+
+export const updateStaff = async (
+  firebaseUID: string,
+  update: { key: string; value: string },
+) => {
+  const updateObj = { $set: { [update.key]: update.value } };
+  const staff = await StaffModel.findOneAndUpdate({ firebaseUID }, updateObj);
+  if (!staff) throw new Error("Staff not found");
```

Comment:
It is better to send descriptive errors using the HTTPErrors class that can be found in the src/utils folder. I recommend following the same format for error handling as in the events.controller file for all methods in this file.